config/executor/config.yaml

```diff
@@ -36,10 +36,6 @@ kubernetes:
   impersonateUsers: false
   trackedNodeLabels:
     - kubernetes.io/hostname
-  etcd:
```

Comment:
Need to provide the metric urls anyway so we may as well leave this empty by default.